source/errors.html.md.erb

```diff
@@ -0,0 +1,85 @@
+---
+title: Errors
+weight: 80
+---
+
+# Errors
+
+This page shows examples of error responses.
+
+Errors will be structured as such
+
+```
+{
+  "errors": [
+    {
+      "status": "422",
+      "title":  "Invalid Attribute",
+      "detail": "Volume does not, in fact, go to 11."
+    }
+  ]
+}
+```
```

Comment:
In this way is not easy to identify the field name (title for example).
what about this something like:
```
{
  "errors": [
    {
      "status": "422",
      "field": "title",
      "error_message":  "Invalid Attribute",
      "detail": "Volume does not, in fact, go to 11."
    }
  ]
}
```

Reply:
the format supported by OpenAPI should be something like:
```
  schemas:
    # Schema for error response body
    Error:
      type: object
      properties:
        code:
          type: string
        message:
          type: string
      required:
        - code
        - message
```
see https://swagger.io/docs/specification/describing-responses/